src/main/java/com/gendeathrow/morechickens/modHelper/MoreRefinedStorageAddon.java

```diff
@@ -0,0 +1,41 @@
+package com.gendeathrow.morechickens.modHelper;
+
+import java.util.List;
+
+import com.setycz.chickens.handler.SpawnType;
+import com.setycz.chickens.registry.ChickensRegistryItem;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+
+public class MoreRefinedStorageAddon extends BaseModAddon
+{
+
+	public static ChickensRegistryItem crystalizedChicken = null;
+
+	public MoreRefinedStorageAddon()
+	{
+		super("morerefinedstorage", "More Refined Storage", "textures/entity/morerefinedstorage/");
+		this.setNeedsModPresent(false);
+	}
+
+	@Override
+	public List<ChickensRegistryItem> registerChickens(List<ChickensRegistryItem> allChickens) {
+
+		crystalizedChicken = addChicken(allChickens,
+				"crystalizedchicken",
+				this.nextID(),
+				"crystalized_chicken.png",
+				this.getFirstOreDictionary("CrystalizedNugget"),
```

Comment:
Not sure you understand what the Ore Dictionary is. Things like `ingotCopper` are in the Ore Dictionary. This means that if a mod adds its own Copper Ingot, it can register it to the Ore Dictionary which allows it to be used interchangeably with Copper Ingots added by other mods.

Using the Ore Dictionary for specifying recipe stuff is so that there isn't a dependency on a specific mod meaning that mods can be interchangable. `CrystalizedNugget` doesn't follow the convention for Ore Dictionary entries which makes me think it's not even an Ore Dictionary entry.